[beta3] Build faills

The beta3 build fails with:

> In file included from /usr/include/string.h:638:0,
>                  from /usr/include/glib-2.0/glib/gtestutils.h:30,
>                  from /usr/include/glib-2.0/glib.h:82,
>                  from /builddir/build/BUILD/gvmd-8.0-beta3/src/gmp_base.h:29,
>                  from /builddir/build/BUILD/gvmd-8.0-beta3/src/gmp_tickets.h:29,
>                  from /builddir/build/BUILD/gvmd-8.0-beta3/src/gmp_tickets.c:33:
> In function 'memset',
>     inlined from 'create_ticket_reset' at /builddir/build/BUILD/gvmd-8.0-beta3/src/gmp_tickets.c:351:10:
> /usr/include/bits/string3.h:84:3: error: call to __builtin___memset_chk will always overflow destination buffer [-Werror]
>    return __builtin___memset_chk (__dest, __ch, __len, __bos0 (__dest));
>    ^
> In function 'memset',
>     inlined from 'modify_ticket_reset' at /builddir/build/BUILD/gvmd-8.0-beta3/src/gmp_tickets.c:635:10:
> /usr/include/bits/string3.h:84:3: error: call to __builtin___memset_chk will always overflow destination buffer [-Werror]
>    return __builtin___memset_chk (__dest, __ch, __len, __bos0 (__dest));
>    ^
> In function 'memset',
>     inlined from 'create_ticket_start' at /builddir/build/BUILD/gvmd-8.0-beta3/src/gmp_tickets.c:366:10:
> /usr/include/bits/string3.h:84:3: error: call to __builtin___memset_chk will always overflow destination buffer [-Werror]
>    return __builtin___memset_chk (__dest, __ch, __len, __bos0 (__dest));
>    ^
> In function 'memset',
>     inlined from 'modify_ticket_start' at /builddir/build/BUILD/gvmd-8.0-beta3/src/gmp_tickets.c:650:10:
> /usr/include/bits/string3.h:84:3: error: call to __builtin___memset_chk will always overflow destination buffer [-Werror]
>    return __builtin___memset_chk (__dest, __ch, __len, __bos0 (__dest));
>    ^
> cc1: all warnings being treated as errors
> make[2]: *** [src/CMakeFiles/gvmd-pg.dir/gmp_tickets.c.o] Error 1
> make[2]: *** Waiting for unfinished jobs....
> /builddir/build/BUILD/gvmd-8.0-beta3/src/manage_sql.c: In function 'escalate_2':
> /builddir/build/BUILD/gvmd-8.0-beta3/src/manage_sql.c:12810:24: error: 'filter' may be used uninitialized in this function [-Werror=maybe-uninitialized]
>                    body = alert_message_print (message, event, event_data,
>                         ^
> cc1: all warnings being treated as errors
> make[2]: *** [src/CMakeFiles/gvmd-pg.dir/gmp_tickets.c.o] Error 1
> make[2]: *** Waiting for unfinished jobs....
> /builddir/build/BUILD/gvmd-8.0-beta3/src/manage_sql.c: In function 'escalate_2':
> /builddir/build/BUILD/gvmd-8.0-beta3/src/manage_sql.c:12810:24: error: 'filter' may be used uninitialized in this function [-Werror=maybe-uninitialized]
>                    body = alert_message_print (message, event, event_data,
>                         ^
> cc1: all warnings being treated as errors
> make[2]: *** [src/CMakeFiles/gvmd-pg.dir/manage_sql.c.o] Error 1
> make[2]: Leaving directory `/builddir/build/BUILD/gvmd-8.0-beta3'
> make[1]: Leaving directory `/builddir/build/BUILD/gvmd-8.0-beta3'
> make[1]: *** [src/CMakeFiles/gvmd-pg.dir/all] Error 2
> make: *** [all] Error 2


OS: CentOS 7.6

--- a/src/gmp_tickets.c
+++ b/src/gmp_tickets.c
@@ -348,7 +348,7 @@ create_ticket_reset ()
       g_slist_free_1 (create_ticket_data.context->first);
     }
   g_free (create_ticket_data.context);
-  memset (&create_ticket_data, 0, sizeof (get_tickets_t));
+  memset (&create_ticket_data, 0, sizeof (create_ticket_t));
 }
 
 /**
@@ -363,7 +363,7 @@ create_ticket_start (gmp_parser_t *gmp_parser,
                      const gchar **attribute_names,
                      const gchar **attribute_values)
 {
-  memset (&create_ticket_data, 0, sizeof (get_tickets_t));
+  memset (&create_ticket_data, 0, sizeof (create_ticket_t));
   create_ticket_data.context = g_malloc0 (sizeof (context_data_t));
   create_ticket_element_start (gmp_parser, "create_ticket", attribute_names,
                                attribute_values);
@@ -632,7 +632,7 @@ modify_ticket_reset ()
       g_slist_free_1 (modify_ticket_data.context->first);
     }
   g_free (modify_ticket_data.context);
-  memset (&modify_ticket_data, 0, sizeof (get_tickets_t));
+  memset (&modify_ticket_data, 0, sizeof (modify_ticket_t));
 }
 
 /**
@@ -647,7 +647,7 @@ modify_ticket_start (gmp_parser_t *gmp_parser,
                      const gchar **attribute_names,
                      const gchar **attribute_values)
 {
-  memset (&modify_ticket_data, 0, sizeof (get_tickets_t));
+  memset (&modify_ticket_data, 0, sizeof (modify_ticket_t));
   modify_ticket_data.context = g_malloc0 (sizeof (context_data_t));
   modify_ticket_element_start (gmp_parser, "modify_ticket", attribute_names,
                                attribute_values);

